### ISSUE TITLE
ENOTFOUND when running example

### ISSUE THREAD

**Issue by denolfe** [nadavbar/node-spotify-webhelper#2]
Is this package still able to pull information? I get the following error when running the sample code. 

```
{ [Error: getaddrinfo ENOTFOUND taduoualyk.spotilocal.com]
  code: 'ENOTFOUND',
  errno: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'taduoualyk.spotilocal.com' }
```

Thanks

**Comment by TwanoO67** [comment]
Hello,

Same here, just trid the example, and I got this error:

```
  { [Error: unable to verify the first certificate] code: 'UNABLE_TO_VERIFY_LEAF_SIGNATURE' }
```

**Comment by nadavbar** [comment]
Thanks for reporting about this issue, I was able to reproduce it and solve the issue by adding the following line of code before creating the SpotifyWebHelper object:
process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = '0';

I'll update the module code shortly, but you can add this line to your code in the meanwhile as a workaround.

**Comment by nadavbar** [comment]
Fixed in: f76d891
